app/views/simulator/mobile_disabled.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="container-fluid">
+  <div class="row">
+    <div class="col-lg-6 col-md-6 middle-block">
+      <%= image_tag "SVGs/mobile-user.svg",alt: "mobile-user", :class => "signup-cover center-block" %>
+      <div class="welcome-text">
+        <label>Welecome Back, Mobile User</label>
+      </div>
+    </div>
+
+    <div class="col-lg-6 col-md-6 middle-block">
+      <div class="form-container">
+        <h1 class="pb-3">Simulator Disabled</h1>
+        <p>
+        Unfortunately, CirciutVerse simulator is disabled on mobile devices.
+        You can still use it on your computer.
+        We decided to disable it cause it was hard to use on small screens.
+        Sorry for the inconvenience
+        </p>
```

Comment:
Check grammar (e.g. spelling of `Circuitverse`).

Personally I would recommend the following text:
`Sorry, but the CircuitVerse simulator is currently disabled on this device. Please use a computer with a larger screen to continue.`